### PR TITLE
Fix issue #5131

### PR DESCRIPTION
**Pull request by davehope**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: html/includes/api_functions.inc.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ function get_port_stats_by_port_hostname() {
     $hostname  = $router['hostname'];
     $device_id = ctype_digit($hostname) ? $hostname : getidbyname($hostname);
     $ifName    = urldecode($router['ifname']);
-    $stats     = dbFetchRow('SELECT * FROM `ports` WHERE `device_id`=? AND `ifName`=?', array($device_id, $ifName));
+    $stats     = dbFetchRow('SELECT * FROM `ports` WHERE `device_id`=? AND `ifName`=? AND `deleted` = 0', array($device_id, $ifName));
```
Comment (murrant) [inline html/includes/api_functions.inc.php]: Shouldn't this be `!=`?

**Comment by murrant** [comment]
Need to fix your merge conflict on the contributors agreement.

**Comment by davehope** [comment]
Cancelling this PR in favor of a new one. Gave up trying to fix my outdated repo.

**Comment by murrant** [comment]
You might find this useful
http://docs.librenms.org/Developing/Using-Git/

**Pull request closed without merging**